com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.RenderGraph.cs

```diff
@@ -1535,6 +1537,33 @@ void RenderGizmos(RenderGraph renderGraph, HDCamera hdCamera, TextureHandle colo
             return executed;
         }
 
+        class BindResetCameraSizeForAfterPostProcess
```

Comment:
Most if not all other pass data are called XXXPassData so I think we should rename this for consistency.

Reply:
fixed